docs/external_api/file_manager.md

```diff
@@ -273,10 +272,15 @@ GET /server/files/metadata?filename=tools/drill.gcode
 | `first_layer_extr_temp` |  float   | The configured first layer extruder temperature, in Celsius. |
 | `first_layer_bed_temp`  |  float   | The configured first layer bed temperature, in Celsius.      |
 | `chamber_temp`          |  float   | The configured chamber temperature, in Celsius.              |
-| `filament_name`         |   str    | The name of the filament used.                               |
-| `filament_type`         |   str    | The type of filament used, ie: `PLA`.                        |
+| `filament_name`         |   str    | The name(s) of the filament used.                            |
+| `filament_color`        |   str    | The colors(s) of the filament used in #RRGGBB format.        |
+| `extruder_color`        |   str    | The slicer defined extruder color(s) for the print.          |
+| `filament_temp`         |   str    | The base temperature(s) of filament used, in Celsius.        |
+| `filament_type`         |   str    | The type(s) of filament used, ie: `PLA`.                     |
 | `filament_total`        |  float   | The total length filament used in mm.                        |
 | `filament_weight_total` |  float   | The total weight of filament used in grams.                  |
+| `single_extruder_multi_material` | int | Identifies a multimaterial print with single extruder.   |
```

Comment:
I wonder if we can shorten the name of this field and make it more descriptive.  It seems this indicates that the print was sliced with MMU enabled, so I wonder if it would make sense to have this field be named `has_mmu` or `has_se_mmu`.